dpnp/tests/test_histogram.py

```diff
@@ -113,6 +113,7 @@ def test_digitize_error(self):
         bins_np = dpnp.asnumpy(bins_dp)
         with pytest.raises(TypeError):
             dpnp.digitize(x_np, bins_dp)
+        with pytest.raises(TypeError):
             dpnp.digitize(x_dp, bins_np)
```

Comment:
Alternatively that can be replaced by one line:
```suggestion
        assert_raises(TypeError, dpnp.digitize, x_dp, bins_np)
```

Reply:
I just wanted to share alternative way how it can be implemented. Both approaches are equivalent.
I think it is fine to either replace all checks in this test or to leave them unchanged, as they are now.